sdk/textanalytics/azure-ai-textanalytics/tests/test_analyze.py

```diff
@@ -708,3 +708,57 @@ def test_pii_action_categories_filter(self, client):
         assert action_results[1][0].entities[0].text == "111000025"
         assert action_results[1][0].entities[0].category == PiiEntityCategory.ABA_ROUTING_NUMBER
         assert action_results[2][0].entities == []  # No Brazilian CPF since not in categories_filter
+
+    @TextAnalyticsPreparer()
+    @TextAnalyticsClientPreparer()
+    def test_partial_success_for_actions(self, client):
+        docs = [{"id": "1", "language": "tr", "text": "I did not like the hotel we stayed at."},
+                {"id": "2", "language": "en", "text": "I did not like the hotel we stayed at."}]
+
+        response = client.begin_analyze_actions(
+                docs,
+                actions=[
+                    AnalyzeSentimentAction(),
+                    RecognizePiiEntitiesAction(),
+                ],
+                polling_interval=self._interval(),
+            ).result()
+
+        action_results = list(response)
+        assert len(action_results) == len(docs)
+        action_order = [
+            _AnalyzeActionsType.ANALYZE_SENTIMENT,
+            _AnalyzeActionsType.RECOGNIZE_PII_ENTITIES,
+        ]
+
+        assert len(action_results[0]) == len(action_order)
+        assert len(action_results[1]) == len(action_order)
+
+        # first doc
+        assert isinstance(action_results[0][0], AnalyzeSentimentResult)
```

Comment:
I don't get this part. How the partial result is possible here. They are two different actions?

Reply:
This block of code checks the action results of the first document. The sentiment action succeeded so we get the sentiment result. The Pii action failed because it specifies an unsupported language, so we get a DocumentError. The response is a list of lists for python, so we get
```
[[AnalyzeSentimentResult, DocumentError],  <-- first doc
[AnalyzeSentimentResult, RecognizePiiEntitiesResult]]   <-- second doc
```

Reply:
Oh. the unsupported language.